trin-utils/build.rs

```diff
@@ -0,0 +1,9 @@
+use std::process::Command;
+fn main() {
+    let output = Command::new("git")
+        .args(["rev-parse", "HEAD"])
+        .output()
+        .expect("Unable to get git hash");
+    let git_hash = String::from_utf8(output.stdout).unwrap_or_default();
```

Comment:
I think the build should fail if the git output is not valid utf8, instead of returning an empty string:
```suggestion
    let git_hash = String::from_utf8(output.stdout).expect("git rev-parse output must be a valid utf8 encoding");
```